src/utils/geo.js

```diff
@@ -66,14 +67,15 @@ export let geoCodeLocation = async (
     console.error('geoCodeLocation', data.error)
     throw new Error(data.error)
   } else {
-    var position = _.get(
-      'Response.View.0.Result.0.Location.DisplayPosition',
-      data
-    )
-
-    return {
-      latitude: position.Latitude,
-      longitude: position.Longitude,
-    }
+    return _.get('Response.View.0.Result.0', data)
   }
 }
+
+export let geoCodeLocation = async (
+  locationId,
+  hereConfig = defaultHereConfig
+) =>
+  _.flow(
+    _.get('Location.DisplayPosition'),
+    _.mapKeys(_.toLower)
```

Comment:
@daedalus28 HereMaps doesn't return anything else here so this will always return an equivalent two-key object to what the old code returned. We could throw in a _.pick to guarantee that, but it's not needed for the actual HereMaps data

Reply:
nice and concise - works for me 👍